### PR TITLE
Added seeding procedure for TeacherAssignment

### PR DESCRIPTION
**Pull request by keke8273**
To be able to add teachAssignments, seeded the semester table as well.

### CODE REVIEW

File: Pikeman/Pikeman.ExamManagement/ReadModel/Implementation/ExamManagementDbContextInitializer.cs [hunk 1]
```diff
@@ -134,12 +161,43 @@ private static void SeedTeachers(ExamManagmentDbContext context, Assembly assemb
...
+                var @class = context.Classes.Local.FirstOrDefault(x => x.EnrollmentYear == seed.ClassEnrollmentYear
+                                                                       && x.ClassNumber == seed.ClassNumber);
+                var teacher = context.Teachers.Local.FirstOrDefault(x => x.FullName == seed.TeacherName);
+                
+                var subject = context.Subjects.Local.FirstOrDefault(x => x.Name == seed.SubjectName);
+                
+                var teacherAssignment = new TeacherAssignment
+                {
+                    Id = GuidUtil.NewSequentialId(),
+                    Semester =  semester,
```
Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/ReadModel/Implementation/ExamManagementDbContextInitializer.cs]: So we are still using semester at the moment

File: Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv [hunk 2]
```diff
@@ -0,0 +1,63 @@
+﻿ClassEnrollmentYear,ClassNumber,SemesterYear,SemesterType,SubjectName,TeacherName
+2014,1,2014,第二学期,语文,潘虹,
```
Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv]: the SemesterYear is 2015 for 第二学期， remember every student starts from September.

File: Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv [hunk 3]
```diff
@@ -68,4 +68,5 @@
 高挠狮,13664789367,英语,
 高春焕,15184718083,化学,
 高银刚,13848811653,物理,
-乌兰哈斯,13327116855,语文
+乌兰哈斯,13327116855,语文,
+许浩馨,,物理,
```
Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv]: double ,,

Comment (keke8273) [inline Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv]: '' is intentional, which is an empty contact number. That teacher is missed in the teacher list. But used in the teacher assignment list

File: Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv (hunk 2 above)
Comment (keke8273) [inline Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv]: 2014 has a second semester too.

Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv]: for those who enrolled in 2014, their second semester is in 2015

File: Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv (hunk 3 above)
Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv]: good to go

File: Pikeman/Pikeman.ExamManagement/ReadModel/Implementation/ExamManagementDbContextInitializer.cs (hunk 1 above)
Comment (keke8273) [inline Pikeman/Pikeman.ExamManagement/ReadModel/Implementation/ExamManagementDbContextInitializer.cs]: yes we are. I created an issue to flag my concerns. I think we are good for the time being. The reporting logic is looking for teach assignments based on semesters. Dont want to change that now.

File: Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv (hunk 2 above)
Comment (keke8273) [inline Pikeman/Pikeman.ExamManagement/SeedData/teacherAssignments.csv]: fixed. new commit added.

File: Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv (hunk 3 above)
Comment (mylaluna) [inline Pikeman/Pikeman.ExamManagement/SeedData/teachers.csv]: Final Checked, good to go.

**Pull request merged**